default/main.py

```diff
@@ -50,12 +65,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         headers={'x-apikey': user.apikey},
                         ssl=ssl_context
                     ) as resp:
-                    json = await resp.json()
-                    json['api_key'] = user.apikey
+                    json = {'api_key': user.apikey}
+                    json.update(await resp.json())
+                    
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                    signature = sign_asymmetric('virustotal-step-2020', 'global', 'webhook-keys', 'webhook-sign', 1, str(json))
```

Comment:
Shouldn't we use json.dumps(json) to convert to string?
https://stackoverflow.com/questions/4547274/convert-a-python-dict-to-a-string-and-back